lib/Rex/Commands/Iptables.pm

```diff
@@ -306,8 +337,8 @@ sub iptables {
     }
   }
 
-  if ( can_run("iptables") ) {
-    my $output = run "iptables $cmd";
+  if ( can_run($iptables) ) {
```

Comment:
If `_get_executable` returns the full path to the executable (commented below), this check might be simpified or pushed into `_get_executable` fully (also the `else` branch).